src/constants/deviceParameters/index.ts

```diff
@@ -46,3 +49,7 @@ export const ABSOLUTE_DATA_MULTI_CHANNEL = 29;
 
 /** enable absolute value sending for a specific channel */
 export const ABSOLUTE_DATA_ENABLE_MULTI_CHANNEL = 30;
+
+export * as rx2SpreadFactors from './rx2SpreadFactors.js';
```

Comment:
this shouldn't be here

Reply:
in that case path to constants `jooby-codec- constants/deviceParameters/rx2SpreadFactors` will be unavailable, any proposal how to do it?

Reply:
reworked